src/rdkafka_broker.h

```diff
@@ -461,7 +461,9 @@ rd_kafka_broker_t *rd_kafka_broker_controller_async(rd_kafka_t *rk,
                                                     int state,
                                                     rd_kafka_enq_once_t *eonce);
 
-int rd_kafka_brokers_add0(rd_kafka_t *rk, const char *brokerlist);
+int rd_kafka_brokers_add0(rd_kafka_t *rk,
+                          const char *brokerlist,
+                          rd_bool_t adding_bootstrap);
```

Comment:
Not sure what convention to follow. Some variables follow snake_case, some follow camelCase.

Reply:
We follow snake_case in general. We follow camelCase if we are using same class or function as Java impl to make it  easy to reference in future

Reply:
`is_bootstrap_server_list` should be better.

Reply:
Updated